tests/conftest.py

```diff
@@ -3,7 +3,7 @@
 import pytest
 
 from poetry.core.utils._compat import Path
-from tests.utils import tempfile
+from tests._helpers.utils import tempfile
```

Comment:
Probably just move utils.py to `tests/utils/__init__.py`

Reply:
Hiding test helpers inside the `__init__.py` does not seem ideal to me.
Is that common?

An alternative that I like more, is to just rename it to something else like `testutils`